tests/python/TestEntities.py

```diff
@@ -98,13 +98,13 @@ def findEntity(self, thID):
                                     ["threadid"], thID)
 
     def test_runMultipleAdds(self):
-
         simultaneous = 1000;
         thread_arr = []
         for i in range(1,simultaneous):
             thread_add = Thread(target=self.addEntity,args=(i,) )
             thread_add.start()
             thread_arr.append(thread_add)
+            time.sleep(0.002)
```

Comment:
I think we can create a separate issue. What I noticed here is....msync increases how long PMGD takes. That means the reader writer lock has more timeouts and so more threads wait. That somehow seems to be messing with some queue that starts denying newer threads from even getting connected. Needs more exploring which I don't want to hold up this pull request for.

Reply:
Also, one of the things we need to figure now at the VDMS layer is that timeout based locking is bound to cause more failures since threads don't get to wait however long. Do we introduce internal retries or let the client retry? We should of course try to figure out the right values as we run more experiments but just saying.